packages/core/streams/ai-stream.ts

```diff
@@ -50,7 +50,7 @@ export interface AIStreamCallbacksAndOptions {
  * @interface
  */
 export interface AIStreamParser {
-  (data: string): string | void;
+  (data: string, event?: string): string | void;
```

Comment:
Can you use an options array with an `event` property as the 2nd parameter? This would leave more flexibility for future extensions without breaking backward compatibility.

Reply:
I prefer the object 👍